pkg/terraform/aws/terraform_aws.go

```diff
@@ -20,11 +20,13 @@ import (
 )
 
 // SetCloudCredentials sets AWS account credentials defined in .aws dir in user home dir
-func SetCloudCredentials(rootBody *hclwrite.Body, region string) error {
+func SetCloudCredentials(rootBody *hclwrite.Body, awsProfile, region string) error {
 	provider := rootBody.AppendNewBlock("provider", []string{"aws"})
 	providerBody := provider.Body()
 	providerBody.SetAttributeValue("region", cty.StringVal(region))
-	providerBody.SetAttributeValue("profile", cty.StringVal("default"))
+	if awsProfile != constants.AWSDefaultCredential {
```

Comment:
I mean, why the if?

Reply:
fixed

Reply:
actually we need it. What if user sets env variables so profile is not needed for terraform.